src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -33,14 +33,16 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        ShoppingCart cart = shoppingCartService
+                .getByUser(userService.findByEmail(authentication.getName()).get());
```

Comment:
```suggestion
                .getByUser(userService.findByEmail(authentication.getName()).orElseThrow( ... ));
```
lets throw an exception